google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/hql/HQLTest.java

```diff
@@ -1510,28 +1455,13 @@ public void test_hql_collection_expressions_example_7() {
 			List<Phone> phones = entityManager.createQuery(
 				"select p " +
 				"from Phone p " +
-				"where current_date() > key( p.callHistory )", Phone.class )
+				"where current_timestamp() > key( p.callHistory )", Phone.class )
 			.getResultList();
 			//end::hql-collection-expressions-example[]
 			assertEquals(2, phones.size());
 		});
 	}
 
-	@Test
-	public void test_hql_collection_expressions_example_8() {
-		doInJPA( this::entityManagerFactory, entityManager -> {
-			//tag::hql-collection-expressions-example[]
-
-			List<Phone> phones = entityManager.createQuery(
-				"select p " +
-				"from Phone p " +
-				"where current_date() > all elements( p.repairTimestamps )", Phone.class )
```

Comment:
I assume this becomes the ` = ALL ( .. ` ?

Reply:
Yup, correct.